core/src/main/scala/backbone/aws/AmazonSqsOps.scala

```diff
@@ -34,14 +34,18 @@ private[backbone] trait AmazonSqsOps extends AmazonAsync {
     ).map(_ => ())
   }
 
-  def createQueue(name: String)(implicit ec: ExecutionContext): Future[QueueInformation] = {
-    logger.info(s"Creating queue. queueName=$name")
+  def createQueue(params: CreateQueueParams)(implicit ec: ExecutionContext): Future[QueueInformation] = {
+
+    val createRequest = new CreateQueueRequest(params.name)
+    params.kmsKeyAlias.map(key => createRequest.addAttributesEntry("KmsMasterKeyId", key))
```

Comment:
maybe we call ist kmsKeyId because you can put in the alias the id and the arn if the documentation is right. This should be documented.